api/base/settings/defaults.py

```diff
@@ -167,6 +167,9 @@
 
 DEBUG_TRANSACTIONS = DEBUG
 
+JWT_SECRET = 'osf_api_cas_login_jwt_secret_32b'
```

Comment:
Oh, I see now that there is a JWT_SECRET in `website.settings` but not in `api.base.settings`. @icereval What do you think? Should we add this here?